popSimManu.tex

```diff
@@ -430,6 +431,7 @@ \subsection*{Simulation engines}
 especially in the presence of selection.
 However, since many sequence-based measures of population diversity remain nearly unchanged when rescaling in this fashion,
 this practice is effective for many purposes and widely employed.
+%ADK: I think we need a citaiton above for scaling. Maybe https://doi.org/10.1534/genetics.113.156935
```

Comment:
we need a citation here I think

Reply:
We cite that paper two sentences back, so I think it's fine.